plugin.d.ts

```diff
@@ -1,3 +1,5 @@
+import {URL} from 'url';
```

Comment:
@sindresorhus is this idomatic?

Reply:
No, but TS forces us to do it: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/34960